src/api/me.py

```diff
@@ -511,7 +512,9 @@ async def delete_alcohol_form_favourites(
     """
     alcohol_id = validate_object_id(alcohol_id)
     user_id = current_user['_id']
-    await UserFavouritesHandler.delete_alcohol_from_favourites(db.user_favourites, user_id, alcohol_id)
+    if await UserFavouritesHandler.check_if_alcohol_in_favourites(db.user_favourites, user_id, alcohol_id):
```

Comment:
W sumie pytanie czy nie dało się to zrobić `if await UserFavouritesHandler.check_if_alcohol_in_favourites()` zamiast dodatkowo sprawdzać. Mongo przy delete chyba może zwrócić usunięty dokument